doc/nrf/libraries/bin/bt_ll_pcft/LICENSE.txt

```diff
@@ -0,0 +1,32 @@
+Copyright (c) 2021, PACKETCRAFT, INC.
```

Comment:
This is the license text that the License and Service Agreement mandates, so this is correct.